.travis.yml

```diff
@@ -1,8 +1,19 @@
 sudo: required
 
 language: python
-python: 
-  - "2.7"
+
+matrix:
+  include:
+  - python: '2.7'
+    env: COVER=1
+    sudo: required
+    services:
+      - docker
+  - python: '2.7'
+    env: CROSSDOCK=1
+    sudo: required
```

Comment:
I seem to recall that in the matrix for Jaeger repo we didn't have to duplicate the sudo/docker instructions